aws/logs_monitoring/lambda_function.py

```diff
@@ -78,14 +73,7 @@ def datadog_forwarder(event, context):
     transformed = transform(enriched)
     metrics, logs, trace_payloads = split(transformed)
 
-    if DD_FORWARD_LOG:
-        forward_logs(logs)
-
-    forward_metrics(metrics)
-
-    if len(trace_payloads) > 0:
-        forward_traces(trace_payloads)
-
+    forward(logs, metrics, trace_payloads)
```

Comment:
nit: What's the benefit of making this wrapper function? Something about it feels weird to me, partly because all these args are lists and it would be easy to mix up the order 🤔

Reply:
I'm actually refactoring this part because I'm working on adding a retry mechanism to be called if we fail to forward events in the case of failures on the intake side. 
Therefore, I'm putting everything under a single function to be called from the retrier side.

Reply:
also there's a benefit from encapsulating the inner calls inside the module and expose less details outside.